ui-tests/src/test/java/org/openmrs/reference/UsingBackButtonInMergePatientTest.java

```diff
@@ -1,23 +1,23 @@
 package org.openmrs.reference;
 
 import org.junit.Before;
+import org.junit.After;
 import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.openmrs.reference.groups.BuildTests;
 import org.openmrs.reference.helper.TestPatient;
 import org.openmrs.reference.page.*;
 import org.openmrs.uitestframework.test.TestBase;
-import org.junit.*;
```

Comment:
This was calling the entire package which is not necessary since we need only a particular tag.